app/models/tenant.rb

```diff
@@ -137,6 +137,17 @@ def set_quotas(quotas)
     get_quotas
   end
 
+  def used_quotas(options = {:include_retired => true})
+    vms = vm_or_templates.includes(:hardware => :disks).all
+
+    vms.reject! { |vm| vm.host_id.nil? || (options[:include_retired] == false && !vm.retired?) }
```

Comment:
The check `vm.host_id.nil?` is only valid for VMware.  For systems like RHEV-M the VM will not be associated with a host when it is off unless it has a specified host affinity and cloud VMs will have a host either.

Maybe this should be checking `vm.archived? || vm.orphaned?`